src/GeoJson/Feature/Feature.php

```diff
@@ -75,27 +71,22 @@ public function getId()
 
     /**
      * Return the properties for this Feature object.
-     *
-     * @return array|null
      */
-    public function getProperties()
+    public function getProperties(): ?array
     {
         return $this->properties;
     }
 
-    /**
-     * @see http://php.net/manual/en/jsonserializable.jsonserialize.php
-     */
     public function jsonSerialize(): array
     {
         $json = parent::jsonSerialize();
 
         $json['geometry'] = isset($this->geometry) ? $this->geometry->jsonSerialize() : null;
-        $json['properties'] = isset($this->properties) ? $this->properties : null;
+        $json['properties'] = $this->properties ?? null;
```

Comment:
```suggestion
        $json['properties'] = (object) $this->properties ?? null;
```

I don't suppose this edge case will come up in practice, but in the event an application used numeric strings for property names (e.g. "0"), this could result in PHP encoding a JSON array for this field. That could be addressed by ensuring the `$this->properties` array is always cast to an object and removing the code below that replaces an empty array with an empty object.

No worries if you'd rather exclude this from the PR, as I'd consider it a longstanding bug.

Reply:
I believe that this would break BC because this would return an object inside the array. I know the method is not meant to be used only implicitly via `json_encode()` but I'm sure there are at least a few users calling it directly 😂.

I think if someone used this with numeric strings they would have run into trouble anyway, at the latest when using the Feature or FeatureCollection elsewhere. They would then have changed their code or used `json_encode($feature, JSON_FORCE_OBJECT)`. (But that's only what I would have done 😅) 

<img width="531" alt="image" src="https://user-images.githubusercontent.com/67554/159873447-458a899a-fe60-4cb6-9044-3c13678d61a6.png">

Reply:
SGTM. I still think there's value in forcing a cast to `(object)`, but that sounds like something better left to a 2.0 since it could be a BC break.